### PR TITLE
chore(libscap): change missed int32_t to ss_plugin_rc

### PR DESCRIPTION
**Pull request by deepskyblue86**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area build

> /area driver-kmod

> /area driver-ebpf

/area libscap

> /area libsinsp

> /area tests

> /area proposals

**What this PR does / why we need it**:
Change missed int32_t (f48b7bd) to ss_plugin_rc
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by leogr** [comment]
/ok-to-test

@deepskyblue86 good catch :+1:

**Review by leogr — APPROVED** [review]
/approve

File: userspace/libscap/plugin_info.h [hunk 1]
```diff
@@ -359,7 +357,7 @@ typedef struct
 	//   by the plugin and must not be deallocated or modified until the next
 	//   extract_fields() call.
 	//
-	// Return value: An ss_plugin_rc with values SS_PLUGIN_SUCCESS or SS_PLUGIN_FAILURE.
+	// Return value: A ss_plugin_rc with values SS_PLUGIN_SUCCESS or SS_PLUGIN_FAILURE.
```
Comment (gnosek) [inline userspace/libscap/plugin_info.h:360]: This reads to me as "an ess-ess-plugin-rc" so I'd leave the "An" (elsewhere as well). But I'm not going to fight over it so I'll ✅ if you decide "A" is what you prefer.

Comment (deepskyblue86) [reply userspace/libscap/plugin_info.h:360]: I followed the crowd... more "a" than "an" :shrug:

**Review by mstemm — APPROVED** [review]

**Pull request merged**